Python/Product/TestAdapter/PythonRunSettings.cs

```diff
@@ -134,86 +134,22 @@ ref outObj
-                                }
-                            });
-
-                            if (config == null || projInfo == null) {
-                                log.Log(
-                                    MessageLevel.Warning,
-                                    Strings.TestDiscoveryFailedMissingLaunchConfiguration.FormatUI(container.Project)
-                                );
-                                continue;
+                            if (isFirstElement) {
+                                isFirstElement = false;
+                                didFindProjectInfo = WriteProjectInfoForContainer(writer, container, log);
                             }
-                            writer.WriteAttributeString("name", projectName);
-                            writer.WriteAttributeString("isWorkspace", isWorkspace.ToString());
-                            writer.WriteAttributeString("useLegacyDebugger", UseLegacyDebugger ? "1" : "0");
-                            writer.WriteAttributeString("nativeDebugging", nativeCode);
-                            writer.WriteAttributeString("djangoSettingsModule", djangoSettings);
-                            writer.WriteAttributeString("testFramework", testFramework);
-                            writer.WriteAttributeString("workingDir", config.WorkingDirectory);
-                            writer.WriteAttributeString("interpreter", config.GetInterpreterPath());
-                            writer.WriteAttributeString("pathEnv", config.Interpreter.PathEnvironmentVariable);
-                            writer.WriteAttributeString("unitTestRootDir", unitTestRootDir);
-                            writer.WriteAttributeString("unitTestPattern", unitTestPattern);
-
-                            writer.WriteStartElement("Environment");
-
-                            foreach (var keyValue in config.Environment) {
-                                writer.WriteStartElement("Variable");
-                                writer.WriteAttributeString("name", keyValue.Key);
-                                writer.WriteAttributeString("value", keyValue.Value);
-                                writer.WriteEndElement();
-                            }
-                            writer.WriteEndElement(); // Environment
 
-                            writer.WriteStartElement("SearchPaths");
-                            foreach (var path in config.SearchPaths) {
-                                writer.WriteStartElement("Search");
-                                writer.WriteAttributeString("value", path);
-                                writer.WriteEndElement();
+                            if (didFindProjectInfo) {
+                                writer.WriteStartElement("Test");
+                                writer.WriteAttributeString("file", container.Source);
+                                writer.WriteEndElement(); // Test    
```

Comment:
I could change "Test" to "TestContainer" this is leftover code from when we used to pass the actually tests